ipn/ipnstore/memory.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ipnstore
```

Comment:
Lacks a package doc. (https://github.com/golang/go/wiki/CodeReviewComments#package-comments)

Reply:
indeed my bad, I added it in `ipn/ipnstore/store.go`, would you prefer something dedicated like `ipn/ipnstore/doc.go`?

Reply:
sounds good, this is irrelevant now that I moved it into `ipn/ipnstate` :ok_hand: